### PR TITLE
Update and rename signing-in.md to signing-in.rst

### PR DESCRIPTION
**Pull request by yangchen1**

### CODE REVIEW

File: source/help/getting-started/signing-in.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
+Signing in
+==========
+
+--------------
+
+This documentation refers to Mattermost v3.0+. For v2.2, see `archived
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: This link is no longer valid as we deleted this doc. Let's remove this line.

File: source/help/getting-started/signing-in.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+forgot my password** on the sign in screen, or contact your System Admin
+for help resetting your password.
+
+GitLab Single-Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When enabled by your System Admin, you can sign in using your GitLab
+account using a one-click sign in option.
+
+Google Single-Sign-On (`Enterprise Edition`_)
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: See the system console documentation for the new way we are marking EE features in the docs

File: source/help/getting-started/signing-in.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+SAML Single-Sign-On (SSO) (`Enterprise Edition`_)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When enabled by your System Admin, you can sign in with your SAML
+credentials. This lets you use the same username and password for
+Mattermost that you use for various other company services. Mattermost
+officially supports `Okta`_ and `Microsoft ADFS`_ as an identity
+provider (IDP) for SAML, but you may use other SAML IDPs as well. Please
+see `documentation` to learn more about configuring SAML for
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: missing link on the word `documentation`?

File: source/help/getting-started/signing-in.rst [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+#. Search for “Mattermost” and click **GET** to download the App for
+   free.
+#. Open Mattermost from your homescreen and input your team and account
+   information to login:
+
+   #. Enter Server URL: This is the web address you go to when you want
+      to access Mattermost. You can find the Server URL by asking your
+      System Admin or by looking at the address bar in a desktop browser
+      tab with Mattermost open. It is in the form
+      ``https://domain.com/``.
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: this should not have a trailing slash or it wont work on iOS

File: source/help/getting-started/signing-in.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+   #. Sign in to Mattermost: This is your account login information as
+      decribed by one of the sign in methods above.
+
+.. _archived documentation: http://docs.mattermost.com/archives/docs-v2.2.html#signing-in
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: doc deleted

File: source/help/getting-started/signing-in.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+#. Search for “Mattermost” and click **INSTALL** to download the App for
+   free.
+#. Open Mattermost from your homescreen and input your team and account
+   information to login:
+
+   #. Enter Server URL: This is the web address you go to when you want
+      to access Mattermost. You can find the Server URL by asking your
+      System Admin or by looking at the address bar in a desktop browser
+      tab with Mattermost open. It is in the form
+      ``https://domain.com/``.
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: remove trailing slash for consistency (even though it works on android)

File: source/help/getting-started/signing-in.rst (hunk 1 above)
Comment (yangchen1) [inline source/help/getting-started/signing-in.rst]: Done.

File: source/help/getting-started/signing-in.rst (hunk 3 above)
Comment (yangchen1) [inline source/help/getting-started/signing-in.rst]: Good catch!

File: source/help/getting-started/signing-in.rst (hunk 6 above)
Comment (yangchen1) [inline source/help/getting-started/signing-in.rst]: Fixed.

File: source/help/getting-started/signing-in.rst (hunk 5 above)
Comment (yangchen1) [inline source/help/getting-started/signing-in.rst]: Removed

File: source/help/getting-started/signing-in.rst [hunk 7]
```diff
@@ -0,0 +1,125 @@
+Signing in
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: Please also update the index file due to the .md > .rst conversion: https://github.com/mattermost/docs/blob/master/source/guides/user.rst

File: source/help/getting-started/signing-in.rst [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+You can switch between teams you have joined using the **Main Menu**,
+accessed by clicking the three dots in the top header of any team site
+on the server.
+
+Logging Out
+-----------
+
+You can log out from the **Main Menu**, which is accessed by clicking
+the three dots in the top header on the left side of the screen.
+Clicking “Logout” logs you out of all teams that are signed in and open
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: "Logout" > **Logout**

File: source/help/getting-started/signing-in.rst [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+==========
+
+--------------
+
+You can sign in to your team from the root page of your Mattermost site
+``https://domain.com``. After signing in, you will be able to view a
+list of your teams and select which one to open.
+
+Sign In Methods
+---------------
```
Comment (esethna) [inline source/help/getting-started/signing-in.rst]: Heading style needs to be updated to match style guide

File: source/help/getting-started/signing-in.rst (hunk 8 above)
Comment (lfbrock) [inline source/help/getting-started/signing-in.rst]: Should we also update this to just say "logs you out of all teams on the server"?

File: source/help/getting-started/signing-in.rst [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+#. Open Mattermost from your homescreen and input your team and account
+   information to login:
+
+   #. Enter Server URL: This is the web address you go to when you want
+      to access Mattermost. You can find the Server URL by asking your
+      System Admin or by looking at the address bar in a desktop browser
+      tab with Mattermost open. It is in the form
+      ``https://domain.com``.
+   #. Sign in to Mattermost: This is your account login information as
+      decribed by one of the sign in methods above.
```
Comment (lfbrock) [inline source/help/getting-started/signing-in.rst]: decribed > described

File: source/help/getting-started/signing-in.rst [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
```
Comment (lfbrock) [inline source/help/getting-started/signing-in.rst]: decribed > described

File: source/help/getting-started/signing-in.rst (hunk 7 above)
Comment (yangchen1) [inline source/help/getting-started/signing-in.rst]: Done :) Thanks.

File: source/help/getting-started/signing-in.rst (hunk 9 above)
Comment (yangchen1) [inline source/help/getting-started/signing-in.rst]: Done.

**Comment by lfbrock** [comment]
+1

**Pull request merged**